src/slick.core.ts

```diff
@@ -354,6 +374,63 @@ export class SlickRange {
   };
 }
 
+/**
+ * A structure containing a range of cells to copy to.
+ * @class SlickCopyRange
+ * @constructor
+ * @param fromRow {Integer} Starting row.
+ * @param fromCell {Integer} Starting cell.
+ * @param rowCount {Integer} Row Count.
+ * @param cellCount {Integer} Cell Count.
+ */
+export class SlickCopyRange {
+  fromRow: number;
+  fromCell: number;
+  rowCount: number;
+  cellCount: number;
+
+  constructor(fromRow: number, fromCell: number, rowCount: number, cellCount: number) {
+    this.fromRow = fromRow;
+    this.fromCell = fromCell;
+    this.rowCount = rowCount;
+    this.cellCount = cellCount;
+  }
+
+}
+
+/**
+ * Create a handle element for Excel style drag-replace
+ * @class DragExtendHandle
+ * @constructor
+ * @param gridUid {String} string UID of parent grid
+ */
+export class SlickDragExtendHandle {
+  id: string;
+  cssClass = 'slick-drag-replace-handle';
+
+  constructor(gridUid: string) {
+    this.id = gridUid + "_drag_replace_handle";
+  }
+
+  getHandleHtml() {
+    return '<div id="' + this.id + '" class="slick-drag-replace-handle"></div>';
+    //console.log('DragReplaceEl.getStringEl');
+  }
+
+  removeEl() {
+    const dragReplaceEl = document.getElementById(this.id);
+    if (dragReplaceEl) { dragReplaceEl.remove(); }
+    //console.log('DragReplaceEl.removeEl');
+  }
+
+  createEl(activeCellNode: any) {
+    const dragReplaceEl = document.createElement("div");
+    dragReplaceEl.classList.add("slick-drag-replace-handle");
+    dragReplaceEl.setAttribute("id", this.id);
+    activeCellNode.appendChild(dragReplaceEl);
+    console.log('DragReplaceEl.createEl');
```

Comment:
I found that all the failing Cypress tests were because `activeCellNode` could be null  and that caused Cypress to simply fail. The super easy fix is to simply wrap it all in an `if` to make sure it's defined before trying to append to it. 
```suggestion
    if (activeCellNode) {
      const dragReplaceEl = document.createElement("div");
      dragReplaceEl.classList.add("slick-drag-replace-handle");
      dragReplaceEl.setAttribute("id", this.id);
      activeCellNode.appendChild(dragReplaceEl);
    }
```